cypress/e2e/API/EngageSphere.cy.js

```diff
@@ -0,0 +1,151 @@
+/// <reference types="Cypress" />
+describe('Validações de API', () => {
+    context('Solicitações inválidas de forma elegante', () => {
+      it('Então valido page="invalid" como string', () => {
```

Comment:
Que tal algo assim?

```suggestion
      it('Retorna um erro e código de status 400 ao passar um valor inválido para o parâmetro page', () => {
```

Recomendo aplicar a mesma idéia de títulos para todos os outros casos de teste.

Ao ler a descrição do testes, deve ser possível entender o que está sendo testado, assim como o resultado esperado.